tools/run_tests/filter_pull_request_tests.py

```diff
@@ -104,7 +105,7 @@ def add_trigger(self, trigger):
   'config\.m4$': [_PHP_TEST_SUITE],
   'CONTRIBUTING\.md$': [],
   'Gemfile$': [_RUBY_TEST_SUITE],
-  'grpc.def$': [_WINDOWS_TEST_SUITE],
+  'grpc\.def$': [_WINDOWS_TEST_SUITE],
   'grpc\.gemspec$': [_RUBY_TEST_SUITE],
```

Comment:
In that case, would it be best to just remove skipping sanity tests completely to be safe? RIght now, the bottleneck for pull request tests are interop and performance, so we could worry about sanity tests taking too long when the basic tests become the bottleneck.